app/src/main/java/com/jerboa/model/PersonProfileViewModel.kt

```diff
@@ -180,6 +181,20 @@ class PersonProfileViewModel(personArg: Either<PersonId, String>, savedMode: Boo
         }
     }
 
+    fun lockPost(form: LockPost) {
```

Comment:
I have pending changes that deduplicates all these actions. But for now it is good. I'll refactor this later too

Reply:
Sweet, thx.